src/main.cpp

```diff
@@ -1,23 +1,27 @@
+#include <sstream>
+
 #include <seqan3/argument_parser/all.hpp>
 #include <seqan3/core/debug_stream.hpp>
 
+#include "fastq_conversion.hpp"
 #include "give_me_five.hpp"
 
 int main(int argc, char ** argv)
 {
-    seqan3::argument_parser parser{"I-want-five", argc, argv};
+    seqan3::argument_parser parser{"Fastq-to-Fasta-Converter", argc, argv};
 
-    // Declarations for input variables
-    std::vector<size_t> in;
-    std::filesystem::path out;
-    bool yes = false;
+    // Declarations for argument parser
+    std::filesystem::path fastq_file;
```

Comment:
```suggestion
    std::filesystem::path fastq_file{};
```
(to be consistent with the line below)